aixtend/modules/benchmark.py

```diff
@@ -0,0 +1,154 @@
+__author__='shreyassharma'
+
+"""
+Copyright 2022 The aiXplain SDK authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Author: Duraikrishna Selvaraju, Thiago Castro Ferreira, Shreyas Sharma and Lucas Pavanelli
+Date: October 25th 2022
+Description:
+    Benchmark Class
+"""
+import time
+import json
+import requests
+import logging
+import traceback
+import pandas as pd
+from requests.adapters import HTTPAdapter, Retry
+from aixtend.utils.file_handler import download_file
+
+class Benchmark:
```

Comment:
Even though we have functions to create, start and download a benchmark, do we have a method to track a benchmark? Can we follow the same pattern that we in Model and Pipelines?
Here's a functional test depicting the same: https://github.com/aixplain/aiXtend/blob/main/tests/model_test.py